src/components/BookCollection/index.tsx

```diff
@@ -1,21 +1,55 @@
-import { useEffect } from "react";
+import { LoadingButton } from "@mui/lab";
+import { LinearProgress } from "@mui/material";
+import { useEffect, useState } from "react";
 import { useBooksContext } from "../../store/books/BooksContext";
 import BookCard from "../BookCard";
 import "./BookCollection.css";
 
 const BookCollection = () => {
-  const { books, findAllBooks, countAllBooks } = useBooksContext();
+  const { books, allBooksCount, findAllBooks, countAllBooks } = useBooksContext();
+  const [pageNumber, setPageNumber] = useState(1);
+  const [loading, setLoading] = useState(false);
 
   useEffect(() => {
-    findAllBooks?.();
+    findAllBooks?.(0);
```

Comment:
Rename find all books, like it is done in backend. All implies there is no pagination